v3/internal/runtime/desktop/@wailsio/runtime/src/create.ts

```
@@ -23,6 +23,14 @@ export function ByteSlice(source: any): string {
     return ((source == null) ? "" : source);
 }
 
+/**
+ * Date is a creation function that converts RFC3339 strings
+ * (from Go's time.Time JSON marshaling) to JavaScript Date objects.
+ */
+export function Date(source: any): globalThis.Date | null {
```

Biome (2.3.13)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)